src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
 public class MainApp {
+      public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine excavator = new Excavator();
+        Machine truck2 = new Truck();
+        Machine[] machines = new Machine[] {
+                truck, excavator, Bulldozer
+        };
+
+        for (Machine m : machines) {
```

Comment:
according the clean code recommendations, please use the whole word for variable naming. better do not cut name of the variables and methods.